### PR TITLE
Fix issue #66

### LINKED ISSUE #66: Black ribbon in print with Safari
Hmm.. although in the preview Safari says that the ribbon will be read when I print it's black. Do you have any idea what the problem can be?
screen shot 2018-08-08 at 12 07 58 pm
screen shot 2018-08-08 at 12 07 46 pm

### PR DESCRIPTION
**Pull request by warengonzaga**
Reference: https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-print-color-adjust
Fixes: #66 

Just add new class ``printer`` if you want to printer ready your page.

### CODE REVIEW

**Comment by warengonzaga** [comment]
@simonwhitaker can you please review this thank you!

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <title>Fork me on GitHub CSS Ribbon</title>
-    <link rel="stylesheet" href="https://oss.maxcdn.com/libs/normalize-css/3.0.0/normalize.min.css">
+    <link rel="stylesheet" href="https://necolas.github.io/normalize.css/8.0.1/normalize.css">
```
Comment (simonwhitaker) [inline index.html:7]: Where did this CDN link from from? The [README](https://github.com/necolas/normalize.css/blob/master/README.md#normalizecss) links to [yarnpkg.com](https://classic.yarnpkg.com/en/package/normalize.css), which lists a few CDN URLs, but this isn't on the list.

File: gh-fork-ribbon.css [hunk 2]
```diff
@@ -21,6 +21,11 @@
   position: fixed;
 }
 
+/* Fix for Safari when printing */
+.github-fork-ribbon.printer {
```
Comment (simonwhitaker) [inline gh-fork-ribbon.css:25]: Thanks for the contribution! I'm not sure this is the right fix though, for a few reasons:

1. It doesn't work for me (Safari 14.0 on macOS 10.15.7). Without your fix, the banners have a _white_ background both in the print preview window and when "printed" to PDF. With your fix, the banners show the correct colour in the print preview window, but are black when "printed" to PDF.
2. Why is this a separate class? Why not just make `-webkit-print-color-adjust: exact` the default?
3. If it does need to be a separate class, I'd suggest naming the class more meaningfully. E.g. the purpose of this class is to fix colours when printing from webkit browsers, so something like `webkit-print-colors` would be more self-explanatory.

**Review by simonwhitaker — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (warengonzaga) [reply index.html:7]: I get it on the official website @simonwhitaker [here](https://necolas.github.io/normalize.css/). When you right click it and copy the link address.

File: gh-fork-ribbon.css (hunk 2 above)
Comment (warengonzaga) [reply gh-fork-ribbon.css:25]: 1. got it
2. I suggest we should make it as a class so the user can have options to apply the fix or not. Not suggesting to make it globally default.
3. I prefer shorter one which is "printer" I suppose to name it "support-printer" but does not make sense. What do you suggest?

**Pull request closed without merging**